notebooks/en/code_search.ipynb

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
<p>You can put the NOTE in a special Tip block like this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&gt; [!TIP]
&gt; In-memory can only be used for quick prototyping and tests.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Done in [`f82bf10` (#151)](https://github.com/huggingface/cookbook/pull/151/commits/f82bf10b7fd32c4c26139395fbbce70b7eb6eb00).